src/App.tsx

```diff
@@ -5,7 +5,21 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const handleAdd = (newMovie: Movie) => {
+    setMovies(movie => {
```

Comment:
```suggestion
    setMovies(prevMovies => {
```
Actually, it's an array of movies, not singular movie. It should be clear from name